### PR TITLE
qrcode: add white border & handle bitmap error

### PR DESCRIPTION
**Pull request by lvaccaro**
Add white border around qrcode because most of scanner app are not able to detect it.
Try to improve error handling of qrcode generation

### CODE REVIEW

File: src/html.rs [hunk 1]
```diff
@@ -67,21 +68,24 @@ pub fn to_data_url<T: AsRef<[u8]>>(input: T, content_type: &str) -> String {
...
 /// Creates QR containing `message`
-pub fn create_bmp_qr(message: &str) -> Result<Vec<u8>, std::io::Error> {
+pub fn create_bmp_qr(message: &str) -> Result<Vec<u8>, BmpError> {
     let qr = QrCode::new(message.as_bytes()).unwrap();
 
     // The `.mul(3)` with pixelated rescale shouldn't be needed, however, some printers doesn't
     // recognize it resulting in a blurry image, starting with a bigger image mostly prevents the
     // issue at the cost of a bigger image size.
-    let bmp = qr.to_bmp().mul(4).unwrap();
+    let bmp = qr.to_bmp().mul(4)?.add_white_border(4)?;
```
Comment (RCasatta) [inline src/html.rs:77]: the recommended spacing for QR codes is 4 times the size of a module, it should be `.add_white_border(16)?;`

Comment (lvaccaro) [reply src/html.rs:77]: ah thanks, good to know 16 is better

Comment (lvaccaro) [reply src/html.rs]: updated and rebased

**Comment by notmandatory** [comment]
utACK 97a3635

**Pull request merged**